rails-commenting.rb

```diff
@@ -54,15 +54,15 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # ---8)
+      # ---8) This destroy method will redirect back to the specific blog post if destroy was not succesful. 
       redirect_to blog_post_path(@post)
     end
   end
 
-  # ---9)
+  # ---9) This method is declaring private params for blog_post.
```

Comment:
Close this is creating a private section within the class. any methods in this section are not visible to anything outside of this class.